sql-db/panache-flyway/pom.xml

```diff
@@ -47,6 +51,20 @@
         </dependency>
     </dependencies>
     <profiles>
+        <profile>
+            <!-- Disable native build on this module -->
+            <!-- TODO: https://github.com/quarkusio/quarkus/issues/30515 -->
```

Comment:
I think that is already merged

Reply:
Still failing, see https://github.com/quarkusio/quarkus/issues/30515#issuecomment-1400672476.

Reply:
Native failures fixed by https://github.com/quarkusio/quarkus/pull/30530 and https://github.com/quarkusio/quarkus/pull/30569.